core/src/main/java/org/pac4j/jax/rs/filters/SecurityFilter.java

```diff
@@ -56,7 +54,7 @@ protected void filter(JaxRsContext context) throws IOException {
         // For the later, we interpret the error and abort the request using jax-rs
         // abstractions
         buildLogic(config).perform(context, sessionStore, config, new SecurityGrantedAccessOutcome(), adapter(config),
-                clients, authorizers, matchers, multiProfile);
```

Comment:
I didn't see `multiProfile` as one of the parameters in [SecurityLogic interface](https://github.com/pac4j/pac4j/blob/pac4j-parent-5.7.1/pac4j-core/src/main/java/org/pac4j/core/engine/SecurityLogic.java#L31-L32) so I went ahead and deleted it. 

It seems to me this was missed in the upgrade to pac4j v5 last time. So I can make a separate PR for this if needed.